p2p/libp2p/netMessenger.go

```diff
@@ -258,15 +259,22 @@ func setupExternalP2PLoggers() {
 	}
 }
 
-func createP2PPrivKey(seed string) (*libp2pCrypto.Secp256k1PrivateKey, error) {
-	randReader, err := randFactory.NewRandFactory(seed)
+func createP2PPrivKey(p2pPrivKeyBytes []byte) (libp2pCrypto.PrivKey, error) {
+	if len(p2pPrivKeyBytes) == 0 {
```

Comment:
can you also log when you generate a p2p key?

Reply:
done